### PR TITLE
KH-90: Rename Adult NCD Registration to NCD Medical History

### PR DESCRIPTION
**Pull request by kazlaw**
https://issues.openmrs.org/browse/KH-90

-- @rbuisson do I rename the file as well?

### CODE REVIEW

**Comment by rbuisson** [comment]
@kazlaw , yes please.

**Comment by kazlaw** [comment]
@rbuisson , file has been renamed

File: configs/openmrs_config/ampathforms/ncd_Medical_History.json [hunk 1]
```diff
@@ -0,0 +1,438 @@
+{
+    "name": "ប្រវត្តជំងឺ(NCD Medical History)",
```
Comment (rbuisson) [inline configs/openmrs_config/ampathforms/ncd_Medical_History.json:2]: Add a space in between the 2 strings
```ប្រវត្តជំងឺ (NCD Medical History)```

**Comment by kazlaw** [comment]
> I'm not sure I understand. Did you add a new file and not removed the old one? Or am I missing something?

I renamed the existing one

**Pull request closed without merging**